app/javascript/components/editor/FileEditor.tsx

```diff
@@ -100,6 +100,10 @@ export function FileEditor({
   ) => {
     editorRef.current = editor
 
+    // This fixes a Windows-only issue with the custom code font resulting in the cursor rendering being off
+    // See https://github.com/exercism/website/issues/742#issuecomment-816806513
+    monacoEditor.editor.remeasureFonts()
```

Comment:
I've put this almost at the very top of the `handleEditorDidMount` function to indicate its importance to the user experience and to get it working as soon as possible. @kntsoriano is the right place to put this code?

Reply:
Sounds good!